tutorials/.ipynb_checkpoints/Notebook Corresponding to Training Test-checkpoint.ipynb

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
let's delete the directory called ".ipynb_checkpoints" as the only notebook in this is already in the enclosing folder